sdk/java-sdk/src/main/scala/kalix/javasdk/impl/GrpcClients.scala

```diff
@@ -107,8 +107,12 @@ final class GrpcClients(system: ExtendedActorSystem) extends Extension {
   /** Local gRPC clients point to services (user components or Kalix services) in the same deployable */
   private def getLocalGrpcClient[T](serviceClass: Class[T]): T = {
     proxyHostname match {
-      case Some("localhost") => getGrpcClient(serviceClass, "localhost", selfPort.getOrElse(9000), localAddHeader)
-      case Some(selfName)    => getGrpcClient(serviceClass, selfName, 80, localAddHeader)
+      case Some("localhost") => {
+        getGrpcClient(serviceClass, "localhost", selfPort.getOrElse(9000), localAddHeader)
+      }
+      case Some(selfName) => {
+        getGrpcClient(serviceClass, "localhost", selfPort.getOrElse(8013), localAddHeader)
```

Comment:
I think it would be better if this decision was moved to the proxy and handed to the SDK through the `ProxyInfo` message (defined in discovery.proto in the proxy)

Reply:
(just like hos the proxyHostName is today)

Reply:
Didn't see that coming. Thanks!